systemd/ua-timer.service

```diff
@@ -1,7 +1,6 @@
 [Unit]
 Description=Ubuntu Advantage Timer for running repeated jobs
 After=network.target network-online.target systemd-networkd.service ua-auto-attach.service
-Wants=ua-auto-attach.service
```

Comment:
I understand the idea, but it would not work like that because the `ua-auto-attach.service` is a oneshot.
So, instead of ensure _it is running_,  the `Wants=` there ensures _it runs_. With the directive, the service itself is trying to auto-attach even before the timer service runs, and then tries again (but checking all the variables we set there).

Reply:
Without the wants, the auto-attach is initiated only by the timer service.

Reply:
About the lock, the `After=` clause is already telling the timer to wait for the `ua-auto-attach.service` before it runs, so we're keeping it here.

Reply:
Thanks for this detail @renanrodrigo and your explanation in standup and I confirmed repeated ua-auto-attach behavior as well on every timer tick and the 'fix' without Wants= to only run once. Turns out I misunderstood how the repeated timer triggers work w.r.t. oneshot services.